docs/assets/scss/_elements.scss

```
@@ -15,6 +15,15 @@ img.center {
   margin-left: auto;
   margin-right: auto;
 }
+// A centered screenshot sized to a fraction of the column (e.g. the DCO sign-off
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)